pkg/scripts/node.go

```diff
@@ -51,6 +51,21 @@ var (
 			sleep 10
 		fi
 	`)
+	ensureRestartKubeAPIServerCrictlTemplate = heredoc.Doc(`
+		apiserver_id=$(sudo crictl ps --name=kube-apiserver -q)
+		[ -z "$apiserver_id" ] && exit 1
+		sudo crictl rm "$apiserver_id"
+		sleep 30
+		
+	`)
+
+	ensureRestartKubeAPIServerDockerTemplate = heredoc.Doc(`
+		apiserver_id=$(sudo docker ps --filter="name=k8s_kube-apiserver" -q)
+		[ -z "$apiserver_id" ] && exit 1
+		sudo docker rm -f "$apiserver_id"
+		sleep 30
+		
+	`)
```

Comment:
Please unify those with previously defined that had "script-time" condition, and render that condition depending on provided runtime boolean argument.